src/v/coproc/event_listener.cc

```diff
@@ -29,6 +29,7 @@
 #include <seastar/core/sleep.hh>
```

Comment:
no need to change anything, but the body of the commit message should not depend on other context (even the subject). for example in `This way a periodic evaluation...` the `this` should appears to be depending on the subject.